Doc/library/sys.monitoring.rst

```diff
@@ -180,9 +188,20 @@ The local events are:
 * :monitoring-event:`LINE`
 * :monitoring-event:`INSTRUCTION`
 * :monitoring-event:`JUMP`
-* :monitoring-event:`BRANCH`
+* :monitoring-event:`BRANCH_LEFT`
+* :monitoring-event:`BRANCH_RIGHT`
 * :monitoring-event:`STOP_ITERATION`
 
+Deprecated event
+''''''''''''''''
+
+* ``BRANCH``
+
+The ``BRANCH`` event is deprecated in 3.14.
+Using :monitoring-event:`BRANCH_LEFT` and :monitoring-event:`BRANCH_RIGHT`
+events will give much better performance as they can be disabled
+independently.
```

Comment:
If there is no meaning to "right" and "left" then what does it mean to disable one but not the other?

Reply:
The reason to disable one and not the other: when coverage sees a LEFT event, it doesn't need to hear about it anymore, but it still needs to hear about RIGHT, and vice-versa.